cmd/oras/root/pull.go

```diff
@@ -237,6 +239,10 @@ func runPull(ctx context.Context, opts pullOptions) error {
 	// Copy
 	desc, err := oras.Copy(ctx, src, opts.Reference, dst, opts.Reference, copyOptions)
 	if err != nil {
+		if strings.Contains(err.Error(), "path traversal disallowed") {
```

Comment:
This should not be done via string comparing, oras-go exported a specific type `ErrPathTraversalDisallowed`, see https://pkg.go.dev/oras.land/oras-go/v2@v2.2.0/content/file#pkg-variables

Reply:
Fixed the issue as errors.is